orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/RunTaskHandler.kt

```diff
@@ -232,14 +232,10 @@ class RunTaskHandler(
   private fun RunTask.withTask(block: (StageExecution, TaskExecution, Task) -> Unit) =
     withTask { stage, taskModel ->
       try {
-        taskResolver.getTask(taskModel.implementingClass)
```

Comment:
This call to lookup based on the implementing class name was introduced to support the now deleted `SimpleTask` pattern.  The problem with doing this is there cases wherein this will always fail (like when tasks are nested classes inside stages, the lookup by implementing class name will fail), thus needlessly re-triggering a rebuild of the task cache.